### PR TITLE
feat(core): Status bot

### PR DESCRIPTION
**Pull request by ffflorian**
This bot sends a short message after every successful deployment of `wire-web-packages`.

### CODE REVIEW

File: packages/core/src/demo/status-bot.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+//@ts-check
+
+const APIClient = require('@wireapp/api-client');
+const {Account} = require('@wireapp/core');
+const {Config} = require('@wireapp/api-client/dist/commonjs/Config');
+const {MemoryEngine} = require('@wireapp/store-engine');
+
+const {version} = require('../../package.json');
+const filename = require('path').basename(process.argv[1]);
```
Comment (bennyn) [inline packages/core/src/demo/status-bot.js]: const [a, b] = ...

File: .eslintrc.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ module.exports = {
     "no-magic-numbers": [
       2,
       {
-        "ignore": [-1, 0, 1]
+        "ignore": [-1, 0, 1, 2]
```
Comment (lipis) [inline .eslintrc.js]: Nooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo :/

File: packages/core/src/demo/status-bot.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+//@ts-check
+
+const APIClient = require('@wireapp/api-client');
+const {Account} = require('@wireapp/core');
+const {Config} = require('@wireapp/api-client/dist/commonjs/Config');
+const {MemoryEngine} = require('@wireapp/store-engine');
+
+const {version} = require('../../package.json');
+
+const conversationId = process.argv[2];
```
Comment (lipis) [inline packages/core/src/demo/status-bot.js]:
```
CONVERSATION_ARGUMENT_INDEX = 2;
```

Comment (ffflorian) [reply packages/core/src/demo/status-bot.js]: Please discuss this with @bennyn.

Comment (lipis) [reply packages/core/src/demo/status-bot.js]: The only discussion is what would be the best name for this number 2 :)

**Comment by bennyn** [comment]
@lipis The "2" has been removed.

**Comment by lipis** [comment]
I was already approving it :)

**Pull request merged**